docs/conf.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 Canonical Ltd
```

Comment:
```suggestion
# Copyright 2021 Canonical Ltd.
```

Reply:
Will normalize these for all files in the next PR.